File: Atividade_reforço/Exercícios_C/1.c
#include <stdio.h>
int main ()
{
    char nm[100];
    float sal, t;

    printf("Digite o seu nome: ");
    scanf("%99s", nm);
    printf("Digite seu salário: ");
    scanf("%f", &sal);

    if (sal<=400){
        t=1.15;
    }
    else if (sal<=700){
        t=1.12;
    }
    else if (sal<=1000){
        t=1.10;
    }
    else if (sal<=1800){
        t=1.07;
    }
    else if (sal<=2500){
        t=1.04;
    }
    else{
        t=1;
    }
    printf ("Nome do funcionário: %s\nAumento: %.2f%%\nSalário atual: R$%.2f\nNovo salário: R$%.2f", nm, (t-1)*100, sal, sal*t);    

    return 0;
}

File: Atividade_reforço/Exercícios_python/1.py
nm = str(input("Digite seu nome: "))
sal = float(input("Digite seu salário: "))

if sal<=400:
    t = 1.15
elif sal<=700:
    t = 1.12
elif sal<=1000:
    t = 1.10
elif sal<=1800:
    t = 1.07
elif sal<=2500:
    t = 1.04
else:
    t = 1

print("Nome do funcionário: ", nm, "\nAumento: ", round((t-1)*100, 2), "%", "\nSalário atual: R$", sal, "\nNovo salário: R$", round(sal*t, 2))
